pytorch/training/docker/1.11/py3/cu113/Dockerfile.gpu

```diff
@@ -333,7 +337,7 @@ CMD ["/bin/bash"]
 FROM common AS sagemaker
 
 LABEL maintainer="Amazon AI"
-LABEL dlc_major_version="1"
+LABEL dlc_major_version="2"
```

Comment:
this is in sagemaker, this should not change to 2, as we have not released 1 yet. There can be dlc_major_version 2 in the e3 portion